src/search/screens/search.screen.js

```diff
@@ -15,25 +20,54 @@ import {
 import styled from 'styled-components';
 import { colors, fonts, normalize } from 'config';
 import { isIphoneX, translate } from 'utils';
-import { searchRepos, searchUsers } from '../index';
-
-const mapStateToProps = state => ({
-  users: state.search.users,
-  repos: state.search.repos,
-  locale: state.auth.locale,
-  isPendingSearchUsers: state.search.isPendingSearchUsers,
-  isPendingSearchRepos: state.search.isPendingSearchRepos,
-});
-
-const mapDispatchToProps = dispatch =>
-  bindActionCreators(
-    {
-      searchRepos,
-      searchUsers,
-    },
-    dispatch
+import { RestClient } from 'api';
+
+const NAV_QUERY_PARAM = 'q';
+const SearchTypes = {
+  REPOS: 0,
+  USERS: 1,
+};
+
+const mapStateToProps = (state, ownProps) => {
+  const {
+    auth: { locale },
+    pagination: { SEARCH_REPOS, SEARCH_USERS },
+    entities: { repos, users },
+  } = state;
+
+  const searchQuery = ownProps.navigation.getParam(NAV_QUERY_PARAM);
+
+  const reposSearchResultsPagination = SEARCH_REPOS[searchQuery] || {
+    ids: [],
+    isFetching: false,
```

Comment:
contrarily to other PRs, when we're loading this screen, we're not fetching yet, so we set `isFetching` to false.